client/src/main/java/io/pravega/client/control/impl/Controller.java

```diff
@@ -364,6 +364,14 @@
      */
     CompletableFuture<Transaction.Status> checkTransactionStatus(final Stream stream, final UUID txId);
 
+    /**
+     * Method to get transaction list in completed state(COMMITTED, ABORTED).
+     *
+     * @param stream Stream Name
+     * @return list of transaction ids
+     */
+    CompletableFuture<List<UUID>> listCompletedTransactions(final Stream stream);
```

Comment:
Since we don;t accept txnStatus as a parameter to this API now, we must return the status of the txn in the response. Also please use ContinuationTokenIterator in the response similar to other list APIs.
So API definition in the interface would look something like:
```
AsyncIterator<TxnInfo> listCompletedTxns(final Stream stream)
```

Reply:
Now response have txn status along with txnId.